### ISSUE TITLE
Can I change the "spin animation" globally?

### ISSUE THREAD

**Issue by Arthraim** [mutualmobile/MMProgressHUD#35]
I tried something like this
```objective-c
[MMProgressHUD sharedHUD].hud.animationImages = @[/*...*/];
```
then I realized every time I called `showWithTitle:status:confirmationMessage:cancelBlock:images:` would wipe out the value of `animationImages` property in MMProgressHUD instance which would also be used for evaluating `animationImages` property in  `[MMProgressHUD sharedHUD].hud` as well.

So is that possible for me to use a class method (or appearance) to change the animationImages globally, even if I only call `show`, the images would be changed?

**Comment by larsacus** [comment]
I don't currently have support for globally changing the animation images being used. All customization of that property goes through the class and instance methods like you have already found:

```` objc
+ (void)showWithTitle:(NSString *)title
               status:(NSString *)status
          cancelBlock:(void(^)(void))cancelBlock
               images:(NSArray *)images;
````

With a little bit of refactoring, this could be possible.

**Comment by larsacus** [comment]
Closing this as resolved since #36.
